middleware.js

```diff
@@ -57,8 +57,12 @@ module.exports = function(options) {
     options = { src: options };
   }
 
+  var sassMiddlewareError = null;
+
   // This function will be called if something goes wrong
-  var error = options.error || function() { };
+  var error = options.error || function(err) {
+    sassMiddlewareError = err;
```

Comment:
as mentioned in my comment i actually meant to store the passed instance of `Error` into `sassMiddlewareError`. It sure can be returned, too but i see no use case since this function is only used internally.